internal/provider/nordvpn/connection.go

```diff
@@ -8,7 +8,6 @@ import (
 
 func (p *Provider) GetConnection(selection settings.ServerSelection, ipv6Supported bool) (
 	connection models.Connection, err error) {
-	defaults := utils.NewConnectionDefaults(443, 1194, 0) //nolint:gomnd
-	return utils.GetConnection(p.Name(),
-		p.storage, selection, defaults, ipv6Supported, p.randSource)
+	defaults := utils.NewConnectionDefaults(443, 1194, 51820) //nolint:gomnd
```

Comment:
Does NordVPN support other ports than 51820 for wireguard?

Reply:
only 51820 as far i can tell